src/lib/server/database.ts

```diff
@@ -85,6 +86,7 @@ const genCollection = () => ({
 	nostrReceivedMessages: db.collection<NostRReceivedMessage>('nostr.receivedMessage'),
 	cmsPages: db.collection<CMSPage>('cmsPages'),
 	challenges: db.collection<Challenge>('challenges'),
+	leaderboards: db.collection<Leaderboard>('leaderboards'),
```

Comment:
And index on `beginsAt` should be added (for the code on order payment)

like for challenges